src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Windows.cs

```diff
@@ -66,6 +66,10 @@ public static partial class PlatformDetection
         public static bool IsWindows10Version1803OrGreater =>
             GetWindowsVersion() == 10 && GetWindowsMinorVersion() == 0 && GetWindowsBuildNumber() >= 17134;
 
+        // >= Windows 10 May 2019 Update (19H1)
+        public static bool IsWindows10Version1903OrGreater =>
```

Comment:
Would it be better to call the property IsWindows19H1OrGreater? I know we didn't use to name it like that, but it would be more useful I think.

Reply:
I think using the official versions, rather than the codenames is better. We didn't call the other ones Redstone or Theshold

Reply:
Looks like we still use the "1903" moniker in the version string.

![19h1](https://user-images.githubusercontent.com/1746272/55898215-fbc87e00-5b76-11e9-95f8-6d2800de8129.png)

Reply:
Holy screenshot size, Batman!

Reply:
I know we didn't call the other as RS/TH but I don't think using the version number is better. why you think so? the code will have the version number anyway for the check. but usually the usage would be more interested on the product version and not really the build version.

Reply:
Because the versioning scheme they've moved to is relatively straight forward and is what they are still using publicly (yymm). It is also what other resources (such as Wikipedia, blog posts, and news articles) are referring to it as; after it is given an official name.

Reply:
yes, but I had every time to look it up to know which version match which product. anyway, I am not pushing for that name more than thinking it is more convenient.